internal/it/util.go

```diff
@@ -384,3 +384,16 @@ func getDefaultClient(config *hz.Config) *hz.Client {
 	}
 	return client
 }
+
+// Eventually asserts that given condition will be met in 2 minutes,
+// checking target function every 200 milliseconds.
+func Eventually(t *testing.T, condition func() bool, msgAndArgs ...interface{}) {
+	assert.Eventually(t, condition, time.Minute*2, time.Millisecond*200, msgAndArgs)
+}
+
+// Never asserts that the given condition doesn't satisfy in 3 seconds,
+// checking target function every 200 milliseconds.
+//
+func Never(t *testing.T, condition func() bool, msgAndArgs ...interface{}) bool {
+	return assert.Never(t, condition, time.Second*3, time.Millisecond*200, msgAndArgs)
+}
```

Comment:
I did not use them directly because I want to make sure that we emphasize the defaults that we prefer. 
This is what we did on the core hazelcast repo. 90% of the tests are better of with default durations. 

I will add FailNow to the methods.

Reply:
Added FailNow in https://github.com/hazelcast/hazelcast-go-client/pull/622/commits/e30be09d6a9aedef22d0da4914b48345f8293e88